scio-core/src/main/scala/com/spotify/scio/coders/KryoAtomicCoder.scala

```diff
@@ -123,6 +123,7 @@ final private class ScioKryoRegistrar extends IKryoRegistrar {
     k.forClass[LocalTime](new JodaLocalTimeSerializer)
     k.forClass[LocalDateTime](new JodaLocalDateTimeSerializer)
     k.forClass[DateTime](new JodaDateTimeSerializer)
+    k.forSubclass[Schema](new SchemaSerializer)
```

Comment:
One thing to keep in mind is that starting in Avro 1.10, `Schema`s are marked as `Serializable` [in Avro](https://github.com/apache/avro/blob/release-1.10.0/lang/java/avro/src/main/java/org/apache/avro/Schema.java#L87), so (theoretically!) we shouldn't need to do a Kryo override if Avro 1.10+ is used.

We could check for the runtime Avro version, like:

```scala
Try(
  Schema.Parser.class.getPackage().getImplementationVersion()
).foreach { avroVersion =>
  // This could be more elegant, maybe a regex match?
  if (
    avroVersion.startsWith("1.7.") ||
    avroVersion.startsWith("1.8.") ||
    avroVersion.startsWith("1.9.")
  ) {
     k.forSubclass[Schema](new SchemaSerializer)
  }
}
```

@RustedBones, wdyt?